bench/benchmarks_view.js

```diff
@@ -307,7 +307,7 @@ class BenchmarkRow extends React.Component {
                     {this.renderStatistic('(20% trimmed) Mean',
                         (version) => <p>
                             {formatSample(version.summary.trimmedMean)} ms
-                            {version.name === current.name && change}
+                            {change && version.name === current.name}
```

Comment:
This should still evaluate to `change` (rather than a boolean expression).

`{current && version.name === current.name && change}`?

Reply:
🤦‍♂️